### ISSUE TITLE
update ai intro

### ISSUE THREAD

**Issue by jkoschinsky** [GeoDaCenter/reactgeoda#154]
The current chatbot intro message is:
Hello, I'm GeoDa.Ai chatbot! Let's do spatial analysis! Ask me anything about your data.

Let's change the last sentence to a suggestion for how to get the chatbot to know that the data was loaded. E.g. right now, if you load the data (stl_hom) and then ask: "I just loaded the stl_hom data. Can you summarize the data?" There's no response. Helping the user frame the question in a way so the bot realizes the data was loaded would be helpful.

**Comment by lixun910** [comment]
Thanks, Julia! Can you help to check if this has been fixed in geoda.ai/mapland?

**Comment by jkoschinsky** [comment]
looking great!
Would be great to update prompt by asking user which of the available
variables they want to use, e.g. when they click quantile map.
